### PR TITLE
Expose public method to create OC Exporter

### PR DESCRIPTION
**Pull request by ccaraman**
Refactor OC Exporter creation to expose New(Trace/Metrics)Exporter to be use by forks that extend OC Exporter. 

There is no observable functional changes in this service. 
cc @owais @tigrannajaryan

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
@ccaraman to clarify, this is strictly refactoring and making NewMetricsExporter/NewTraceExporter functions public, right? There are no observable functional changes in the service.

**Comment by ccaraman** [comment]
@tigrannajaryan Yes that is correct. I essentially moved the createOCAgentExporter into opencensus.go and fixed the casing for the structure.

File: exporter/opencensusexporter/opencensus.go [hunk 1]
```diff
@@ -67,7 +70,65 @@ const (
 	errAlreadyStopped
 )
 
-func (oce *ocagentExporter) Shutdown() error {
+// NewTraceExporter creates an Open Census trace exporter.
```
Comment (songy23) [inline exporter/opencensusexporter/opencensus.go]: Nit: s/Open Census/OpenCensus (one word)

File: exporter/opencensusexporter/opencensus.go [hunk 2]
```diff
@@ -67,7 +70,65 @@ const (
...
+		if serr != nil {
+			return nil, fmt.Errorf("cannot configure OpenCensus exporter: %v", serr)
+		}
+		exportersChan <- exporter
+	}
+	oce := &ocAgentExporter{exporters: exportersChan}
+	return oce, nil
+}
+
+// NewMetricsExporter creates an Open Census metrics exporter.
```
Comment (songy23) [inline exporter/opencensusexporter/opencensus.go]: Same here.

**Pull request merged**